CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+## v1.6.0
```

Comment:
Should be updated, we've had two patch versions since and an addition to main since the last patch :)